backend/src/routes/api/products.ts

```diff
@@ -67,10 +72,20 @@ export const handler = (router: Router, routesContext: any) => {
           throw e;
         }
       });
-      return res.status(200).send('Product inserted succesfully');
+      const respObject:IResponse = {
+        status:200,
+        data:req.body,
+        message:"Producto insertado correctamente"
+      }
+      return res.status(200).json(respObject);
     } catch (e) {
       log.error(e);
-      return res.status(500).send('Something went wrong');
+      const respObject:IResponse = {
+        status:400,
```

Comment:
Status 400 is "not permitted". If something went wrong with the server, the code should be 502.